apps/server/src/modules/deletion-console/deletion-execution.console.ts

```diff
@@ -1,6 +1,6 @@
 import { Command, Console } from 'nestjs-console';
 import { ConsoleWriterService } from '@infra/console';
-import { DeletionExecutionUc } from '../uc';
+import { DeletionExecutionUc } from '../deletion/uc';
```

Comment:
```suggestion
import { DeletionExecutionUc } from '@modules/deletion';
```
!!! Circular Dependency